CondCore/L1TPlugins/src/plugin.cc

```diff
@@ -207,3 +207,16 @@ REGISTER_PLUGIN(L1CaloGeometryRecord, L1CaloGeometry);
 using namespace l1t;
```

Comment:
@mulhearn would you mind remove the namespace here and put it explicitly in the class names? This will save us some work in the Global Tag Manager backend. (thanks @diguida for the catch)